catalog/transformer/catalog-transformer-streaming-lib/src/main/java/org/codice/ddf/transformer/xml/streaming/lib/SaxEventToXmlElementConverter.java

```diff
@@ -78,15 +86,18 @@ public SaxEventToXmlElementConverter() throws UnsupportedEncodingException, XMLS
      */
     public SaxEventToXmlElementConverter toElement(String uri, String localName, Attributes atts)
             throws XMLStreamException {
-
+        Map<String, String> scopedNamespaces = new HashMap<>();
+        scopeOfNamespacesAdded.push(scopedNamespaces);
         /*
          * Use the uri to look up the namespace prefix and append it and the localName to the start tag
```

Comment:
Fixed, is this what you meant? I liked having the same name on the methods because the way the SAX API works is there are three main methods that get called (startElement, endElement, and characters). These each give you a specific set of arguments. My idea was that you could just call `toElement` with the arguments that the SAX event gives you and it would just work, you wouldn't have to specify if it's a start, or end, or whatever, it would just work.